app/controllers/application_controller.rb

```diff
@@ -123,4 +123,11 @@ def user_not_authorized
   def resource_not_found
     render 'errors/not_found', status: 404
   end
+  
+  def member_should_be_active
+    if !current_member.active
+      flash[:error] = 'Your account have been deactivated from this timebank'
+      redirect_to '/select_organization'
```

Comment:
better to use the route helpers generated by Rails, so we avoid hardcoding strings